kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java

```diff
@@ -23,11 +23,11 @@
   public static ChunkInfo fromSnapshotMetadata(SnapshotMetadata snapshotMetadata) {
     return new ChunkInfo(
         snapshotMetadata.snapshotId,
-        snapshotMetadata.startTimeUtc,
-        snapshotMetadata.endTimeUtc,
-        snapshotMetadata.startTimeUtc,
-        snapshotMetadata.endTimeUtc,
-        snapshotMetadata.endTimeUtc,
+        snapshotMetadata.startTimeEpochMsUtc,
```

Comment:
If we're going with "epoch" as part of the name we don't need to include "UTC" as well. 

I do like including the units in the name so it's abundantly clear.

Reply:
Good call. I removed the utc from the names.